src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -754,6 +754,33 @@ public void TestProcessorTime()
             Assert.InRange(processorTimeAtHalfSpin, processorTimeBeforeSpin, Process.GetCurrentProcess().TotalProcessorTime.TotalSeconds);
         }
 
+        [Fact]
+        [ActiveIssue(31908, TargetFrameworkMonikers.Uap)]
+        public void TotalProcessorTime_PerformLoop_TotalProcessorTimeValid()
+        {
+            CreateDefaultProcess();
+
+            DateTime startTime = DateTime.UtcNow;
+            TimeSpan processorTimeBeforeSpin = Process.GetCurrentProcess().TotalProcessorTime;
+           
+            // Perform loop to occupy cpu, takes less than a second.
+            int i = int.MaxValue / 16;
+            while (i > 0)
+            {
+                i--;
+            }
+
+            DateTime endTime = DateTime.UtcNow;
+            TimeSpan processorTimeAfterSpin = Process.GetCurrentProcess().TotalProcessorTime;
```

Comment:
Seems like the order of these should be switched.  Otherwise, it could be possible for the total processor time to exceed the number of milliseconds in the time range.

Reply:
Good point, changed it.